src/dbos-executor.ts

```diff
@@ -552,19 +552,15 @@ export class DBOSExecutor {
 
     const wfname = wf.name; // TODO: Should be what was registered in wfInfo...
 
-    const span = this.tracer.startSpan(
-      wfname,
-      {
-        status: StatusString.PENDING,
-        operationUUID: workflowID,
-        operationType: OperationType.WORKFLOW,
-        operationName: wInfo?.name ?? wf.name,
-        authenticatedUser: pctx?.authenticatedUser ?? '',
-        authenticatedRoles: pctx?.authenticatedRoles ?? [],
-        assumedRole: pctx?.assumedRole ?? '',
-      },
-      pctx?.span,
-    );
+    const span = this.tracer.startSpan(wfname, {
+      status: StatusString.PENDING,
+      operationUUID: workflowID,
+      operationType: OperationType.WORKFLOW,
+      operationName: wInfo?.name ?? wf.name,
+      authenticatedUser: pctx?.authenticatedUser ?? '',
+      authenticatedRoles: pctx?.authenticatedRoles ?? [],
+      assumedRole: pctx?.assumedRole ?? '',
+    });
```

Comment:
Likewise I assume we don't need to pass the parent context around anymore because startSpan will get it from `getActive()`?